sdk/eventhub/event-hubs/changelog.md

```diff
@@ -1,3 +1,28 @@
+### 2019-08-06 5.0.0-preview.2
+
+#### General
+- The current `Sender` does not have a provision of "awaiting" on sending a message. We need to add handlers on the `Sender` for accepted, rejected, released, modified to ensure whether the message was successfully sent. 
+  Now, we use a new `AwaitableSender` instead of `Sender` which adds the handlers internally and provides an awaitable `send()` operation.
+- Refactor receiver that allows the same amqp link to be shared between streaming and batching receivers.
+- All time related entites have been updated to use milli seconds as the unit of time for consistency.
+- New error InsufficientCreditError is introduced for the scenario where rhea is unable to send events due to its internal buffer being full. This is a transient error and so is treated as retryable.
+- The error OperationTimeoutError was previously mistakenly classified as an AMQP error which is now corrected. Since this can also be a transient error, it is treated as retryable.
```

Comment:
```suggestion
- The error `OperationTimeoutError` was previously mistakenly classified as an AMQP error which is now corrected. Since this can also be a transient error, it is treated as retryable.
```